### PR TITLE
docs(readme): remove mentions to contributor t-shirt

### PR DESCRIPTION
**Pull request by gszr**
### Summary

We now offer digital badges to contributors!

### CODE REVIEW

**Comment by kbarnard10** [comment]
I also have a PR (#12491) waiting for a second approval for this change.

**Review by hanshuebner — APPROVED** [review]

**Comment by gszr** [comment]
> I also have a PR (#12491) waiting for a second approval for this change.

@kbarnard10 I added your changes to my commit and added you as co-author. (The other PR missed a couple of mentions to t-shirt in the doc). Thanks!

**Review by zhongweiy — APPROVED** [review]

**Pull request merged**
